homebridge.json

```diff
@@ -3,8 +3,7 @@
 	"release": "11.2-RELEASE",
 	"artifact": "https://github.com/grimneko/iocage-plugin-homebridge.git",
 	"properties": {
-		"vnet": "on",
-		"boot": "on"
+		"vnet": "on"
```

Comment:
Please change it to `1`. We would like to keep this normalized. All boolean props would be either `0` or `1`. I think i missed it in the earlier PR as well. This would help UI parse boolean props without making assumptions about true/false as this is how iocage normalizes.